src/Akka.Persistence.Hosting/AkkaPersistenceHostingExtensions.cs

```diff
@@ -98,6 +125,10 @@ internal void Build()
             var finalHocon = ConfigurationFactory.ParseString(adapters.ToString())
                 .WithFallback(Persistence.DefaultConfig()); // add the default config as a fallback
             Builder.AddHocon(finalHocon, HoconAddMode.Prepend);
+            
+            // add the health checks if specified
+            if(HealthCheckRegistration != null)
+                Builder.WithHealthCheck(HealthCheckRegistration);
```

Comment:
Add the health checks to the `AkkaConfigurationBuilder` if they've been defined.